mirascope/base/utils.py

```diff
@@ -222,7 +222,8 @@ def wrapper_generator(*args, **kwargs):
         try:
             for attempt in retries:
                 with attempt:
-                    for value in fn(*args, **kwargs):
+                    generator = fn(*args, **kwargs)
```

Comment:
is this necessary for some reason? if so please add a comment

Reply:
it was for debugging, switched back